akka-projection-grpc/src/main/scala/akka/projection/grpc/consumer/scaladsl/GrpcReadJournal.scala

```diff
@@ -291,6 +294,7 @@ final class GrpcReadJournal private (
         case _                  => offset
       })
 
+    val consecutiveFailCounter = new AtomicInteger(0)
```

Comment:
Shared state here depends on the fact that the same source from invoking eventsBySlices is not re-used by separate projections.